src/formatDate.js

```diff
@@ -50,7 +53,68 @@
  */
 
 function formatDate(date, fromFormat, toFormat) {
-  // write code here
+  const prefixYear = (yearBefore, newFormat) => {
+    const yearUpdateFormat
+      = newFormat.find(partDate => partDate.includes('YY'));
+
+    const previousCentury = '19';
+    const currentCentury = '20';
+    const toDetermineCentury = 30;
+
+    if (yearBefore.length === yearUpdateFormat.length) {
+      return yearBefore;
+    }
```

Comment:
Nice condition, but then it's better to call it before creating the constants so that the constants aren't created when they're not needed.